src/main/kotlin/epgx/functions/Comparisons.kt

```diff
@@ -0,0 +1,12 @@
+package epgx.functions
+
+import epgx.ops.AtAtOp
+import org.jetbrains.exposed.sql.Column
+import org.jetbrains.exposed.sql.CustomFunction
+
+/**
+ * Checks if [this] tsvector column is equals to some [tsQuery] query.
+ */
```

Comment:
It doesn't check if it's equals, it checks if it matches
```suggestion
/**
 * Checks if [this] tsvector column is matched by some [tsQuery] query.
 */
```

Reply:
Fixed